docs/docs/api/hooks/useFrameCallback.md

```diff
@@ -15,7 +15,7 @@ useFrameCallback(callback: (frameTimings: FrameTime) => void, autostart = true):
 #### `callback` [Function]
 
 A single worklet function that will be called on every frame update.
-This function recieves a [`FrameTime`](#frametime-object) object as na argument.
+This function receives a [`FrameTime`](#frametime-object) object as na argument.
```

Comment:
```suggestion
This function receives a [`FrameTime`](#frametime-object) object as an argument.
```

Reply:
This line better be perfect, after all these suggestions.